symbolic-base/src/ZkFold/Protocol/Plonkup/Utils.hs

```diff
@@ -21,7 +19,7 @@ import           ZkFold.Prelude                     (iterateN', log2ceiling)
 import           ZkFold.Symbolic.Class              (Arithmetic)
 
 getParams :: forall a . (Ord a, FiniteField a) => Natural -> (a, a, a)
-getParams n = findK' $ mkStdGen 0
+getParams n = findK' 0
```

Comment:
You can use [`mkStdGen64`](https://hackage-content.haskell.org/package/random-1.3.1/docs/System-Random.html#v:mkStdGen64) here instead. My guess is that WASM is 32-bit architecture and, because of it, `mkStdGen` behaves differently (see docs for `mkStdGen64`)